ada_feeding_perception/ada_feeding_perception/food_on_fork2.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3
+
+import rclpy
+from rclpy.node import Node
+
+from sensor_msgs.msg import Image
+from std_msgs.msg import Int8MultiArray
+
+import cv2 as cv
+from cv_bridge import CvBridge, CvBridgeError
+import numpy as np
+
+
+class FoodOnFork(Node):
+    def __init__(self):
+        super().__init__('food_on_fork')
+
```

Comment:
- create a field that has latest_color_img
- update that in the color callback

overlay the color img on the depth img
- otsu thresholding of the depth img 
- https://note.nkmk.me/en/python-opencv-numpy-alpha-blend-mask/